spyder/plugins/editor/widgets/codeeditor.py

```diff
@@ -275,11 +275,11 @@ class CodeEditor(TextEditBaseWidget):
     lsp_response_signal = Signal(str, dict)
 
     #: Signal to display object information on the Help plugin
-    sig_display_object_info = Signal(str)
+    sig_display_object_info = Signal(object)

```

Comment:
I don't understand this change. Why is it needed?

Reply:
I was testing something out, but my question is: The LSP will always return `str` on python 2.7 ?

Reply:
I'm not sure. In that case this change would be good.